packages/mdc-menu/simple/foundation.js

```diff
@@ -149,6 +165,20 @@ class MDCSimpleMenuFoundation extends MDCFoundation {
     this.adapter_.deregisterBodyClickHandler(this.documentClickHandler_);
   }
 
+  /**
+   * @param {Corner} corner Default anchor corner alignment of top-left menu corner.
+   */
+  setAnchorCorner(corner) {
+    this.anchorCorner_ = corner;
+  }
+
+  /**
+   * @param {AnchorMargin} margin 4-plet of margins from anchor.
```

Comment:
hmmm. 4-plet is not really a word...though I see what you are trying to say here. Maybe just add an example of what an argument would look like?

Reply:
Add nullability qualifier to type (e.g., `@param {!AnchorMargin}`

Reply:
tests have examples of how the argument looks like. I added flexibility to specify just the non-zero (default) values.